src/test/java/com/google/step/snippet/W3SchoolsClientTest.java

```diff
@@ -0,0 +1,59 @@
+package com.google.step.snippet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import com.google.step.snippet.data.Card;
+import com.google.step.snippet.external.W3SchoolsClient;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class W3SchoolsClientTest {
+
+  private final W3SchoolsClient client = new W3SchoolsClient();
+
+  @Test
+  public void htmlCodeCard() {
+    Card actual = client.search("https://www.w3schools.com/tags/tag_img.asp");
+    Card expected =
+        new Card(
+            "HTML <img> Tag",
+            "<img src=\"img_girl.jpg\" alt=\"Girl in a jacket\" width=\"500\" height=\"600\">",
+            "https://www.w3schools.com/tags/tag_img.asp",
+            "How to insert an image:");
+    assertEquals(actual, expected);
```

Comment:
http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)

The order is actually `assertEquals(expected, actual)`. This only matters to make the error message read correctly. (and so forth for the others)